sdk/go/auto/workspace.go

```diff
@@ -78,7 +78,7 @@ type Workspace interface {
 	// PulumiVersion returns the version of the underlying Pulumi CLI/Engine.
 	PulumiVersion() string
 	// WhoAmI returns the currently authenticated user.
-	WhoAmI(context.Context) (string, error)
+	WhoAmI(context.Context) (WhoAmIResult, error)
```

Comment:
As @lukehoban pointed out in https://github.com/pulumi/pulumi/issues/10356, this is a breaking change for `Go`. Just highlighting that.